README.md

```diff
@@ -1,2 +1,17 @@
 # sourcerer-app
 Sourcerer app makes engineering profiles from their git repositories.
```

Comment:
We need consistent messaging. The 'hashing' language is what we decided to adopt across the product.